From: Vincent Bernat <vincent@bernat.im>
Date: Sun, 8 Oct 2017 18:32:44 +0200
Subject: systemd: specify documentation location

Dunno if people actually uses that, systemd does it and Debian linter
tool bugs me about this. :)
---
 etc/systemd/exabgp.service | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/etc/systemd/exabgp.service b/etc/systemd/exabgp.service
index 3a5146f..4c2417b 100644
--- a/etc/systemd/exabgp.service
+++ b/etc/systemd/exabgp.service
@@ -1,5 +1,8 @@
 [Unit]
 Description=ExaBGP
+Documentation=man:exabgp(1)
+Documentation=man:exabgp.conf(5)
+Documentation=https://github.com/Exa-Networks/exabgp/wiki
 After=network.target
 ConditionPathExists=/etc/exabgp/exabgp.conf
 
